recipes/ltl/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "ltl" %}
+{% set version = "2.0.19" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://www.as.utexas.edu/~drory/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f2f37220edd6aff55c9420745daf515e7f447b7a19a5f1250fe95d476384261
+  patches:
+    - 0001_Build_shared_lib.patch
+    - 0002_Fix_test_typo.patch
+    - 0003_Fix_blas_retun_type.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ name }}
+  ignore_run_exports:
+    # Both LAPACK and BLAS are needed during compilation for the test suite,
+    # but are not actually linked against in the LTL library (they are only
+    # needed in header files which are not compiled).
+    # If a downstream package actually needs the LTL marray/blas.h or
+    # marray/lapack.h headers, they should add libblas and / or liblapack to
+    # their requirements/host section.
+    - liblapack
+    - libblas
```

Comment:
I'm not a 100% sure this is the correct way to go (I'm no C/C++/shared lib expert), but it seemed like a good solution.